lib/fedora_migrate/repository_migrator.rb

```diff
@@ -52,6 +37,32 @@ def get_source_objects
 
     private
 
+    def migrate_object source
```

Comment:
Before, this method would migrate an object and there were three different return values:
* `error_message(e)` if an exception was raised
* `@failed` + 1 if no exception was raised but `result` was any value other than `true` (and in so doing logged a warning)
* `true` if the migration succeeded.

Now you return the result of the migration unless an error is raised. Is that :ok:?